[easy] Rename the package name of sqlalchemy-enum-tables

ttung opened the pull request:
### Description
Some tools choke on package names that have spaces.  Rename this accordingly.

### Test plan
None.

phoenixAja commented on third-party/sqlalchemy-enum-tables/setup.py:
```diff
@@ -11,7 +11,7 @@ def reqs():
 
 st.setup(
-	name = "SqlAlchemy Enum Tables",
+	name = "SqlAlchemy-Enum-Tables",
```
is this normal python package name formatting? I thought it would be like `sqlalchemy_enum_tables`

ttung replied:
I can do that.

phoenixAja replied:
haha ok, but not an emergency

ttung replied:
Fixed!

phoenixAja replied:
if i could approve again i would haha

Commit:
Rename the package name of sqlalchemy-enum-tables

### Description
Some tools choke on package names that have spaces.  Rename this accordingly.

### Test plan
None.
